eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Deleg.hs

```diff
@@ -285,7 +289,8 @@ delegationTransition = do
           u1 = Set.singleton hk UM.⋪ Rewards u0
           u2 = Set.singleton hk UM.⋪ Delegations u1
           u3 = Ptrs u2 UM.⋫ Set.singleton hk
-      pure ds {dsUnified = u3}
+          (_doSomethingWithMeSomeday, newDState) = refundKeyDeposit hk (ds {dsUnified = u3})
```

Comment:
So the first of the pair is the value that is being refunded, right? Just curious if there is a particular thing to do with it that you were thinking of